make.js

```diff
@@ -41,8 +42,13 @@ target.build = function() {
 
 target.units = function() {
     target.build();
-
-    pushd('test');
+    var nodeVer = process.versions.node;
+    if(semver.lt(nodeVer,'8.0.0')){
```

Comment:
Could you add some comments to the code regarding why we are checking the Node version?

Reply:
Sure